src/components/Editor/index.js

```diff
@@ -160,16 +164,35 @@ class Editor extends Component {
     } else {
       this.undoTimestamp = 0
     }
+
+    if (evt.keyCode === 27 && !this.state.tabGuarded) {// Esc Key
+      this.setState({tabGuarded:true})
+    }else if(!(evt.shiftKey || evt.keyCode === 27 || evt.keyCode === 9) && this.state.tabGuarded){
```

Comment:
Which I'm thinking shift+tab may be broken in the newest implementation not gonna lie I expect this out of event handlers.  They have that wonderful ability to get tangled up like earbuds if you aren't careful with your approach.  Kind of why I originally wrapped the whole thing in a blocking div.  The past trauma of dealing with edge cases was giving me flashbacks.  I'll dig a little bit further later this evening.